acm/src/main/java/org/literacybridge/acm/gui/assistants/PromptsImport/PromptImportedPage.java

```diff
@@ -127,6 +131,42 @@ protected void onPageEntered(boolean progressing) {
             .filter((item) -> item.getLeft().isImportable())
             .collect(Collectors.toList());
 
+        AudioImporter importer = AudioImporter.getInstance();
+        MetadataStore store = ACMConfiguration.getInstance().getCurrentDB().getMetadataStore();
+
+        for (PromptMatchable matchable : matches) {
+            // Get data of all previous items
+            PromptTarget left = matchable.getLeft();
+            String id = left.getPromptId();
```

Comment:
It is useful in code like this, where IDs and names are flying around, to add a comment
  String id = left.getPromptId();                 // Like "2"
  String title = left.getPromptFilename(); // Like "paused"

And if those comments are not right for this code snippet, then it needs more commentary, because that's how I read it!